docs/en/get_started/usage.md

```diff
@@ -224,6 +224,32 @@ miles supports customizing data generation (rollout) to various degrees.
 
           - `loss_mask` should be the same length as `response_length`, with `1` for tokens that should be included in the loss calculation and `0` for those that should be masked out.
 
+### External Eval Delegates
```

Comment:
optional nit: shall we very briefly point to examples/eval b/c that seems just an example. (not sure in the future we will move it to main code?)